docs/dev/Demo/Slice Demo Prep/Prime-Demo-Script-Postman-Setup.md

```diff
@@ -0,0 +1,205 @@
+        * [Setting up the Events folder](#setting-up-the-events-folder)
+        * [Setting up the Templates folder](#setting-up-the-templates-folder)
+    * [Running your new collection](#running-your-new-collection)
+
+
+Regenerate with "pre-commit run -a markdown-toc"
+
+<!-- tocstop -->
+## Slice Demo Prep Basics: Playing the Prime:
+During the slice demo there will be various points of handoff from the TOO/TIO to the Prime and vice versa. When playing the prime, we recommend developing visuals that are split up into 3 sections:
+#### Handoff from TOO -> Prime
+    Visual 1: Prime receives a notification of a move. Prime updates the move details. Prime performs another action (e.g. creates a service item)
+#### Handoff from Prime --> TOO 
+After the Prime creates the Service Item(or Reweigh Request/SIT Extension/ or another action), there is a verbal handoff to the TOO. Nothing technical happens.
+#### Handoff TOO --> Prime
+The TOO reviews the Service Item and approves them and there is a verbal handoff to the Prime to show how a payment request is created and proof od service is uploaded
+    
+    Visual 2: Prime creates a payment request
+    Visual 3: uploads proof of service.
+    
+NOTE: You may be able to combine visuals 2 and 3 by nesting the POST requests for creating a payment request and uploading proof of service, but this involves extensive research into [grouping these requests](https://learning.postman.com/docs/sending-requests/intro-to-collections/).
+#### Handoff from Prime to TOO/TIO
+After the prime uploads the proof of service, they submit the payment request ID to the TIO and do a verbal handoff to the TOO.
+
+## Getting Started:
+We have created a repo in GitHub with postman collections from previous slice demos. This means you can get a copy of all of them the way you would clone any other repo on GitHub: 
+
+1. Clone the `prime_api_deliverable` repo: `git clone git@github.com:transcom/prime_api_deliverable.git`
+2. Navigate to the `postman` folder, where you will find previous slice demo collections (e.g. 202108-slice-demo)
+3. Copy over contents from a previous collection or create a new collection from scratch.
+
+If you have never set up postman before checkout our instructions for [setting up postman](https://transcom.github.io/mymove-docs/docs/dev/tools/Postman/Setting-Up-Postman).
+### Creating a Collection from scratch:
+Once you have cloned the `prime_api_deliverable` repo, create a new branch if someone is reviewing your script or just start working directly in master. 
+Open the file in your favorite text editor. Under the postman folder, create a new folder for our slice demo collection.
+Our naming convention for each slice demo folder is going to be: YYYYMM-slice-demo.
+
+If your slice demo is happening in September 2021, for example, you will create a folder called `202109-slice-demo`. In this folder create 3 more empty folders, and a file for payloads:
+* `payloads.js` - feeds the requests folder
```

Comment:
I'm curious about making payloads.js a single file instead of a folder? I noticed in the existing collection in the repo that it's a folder and it houses multiple files since I believe most collections will have multiple payloads associated with them

Reply:
Oh no, I think I need to make that clearer in this section. You're right, it is a single file. I'll make an update to this section.